CLIENTS.md

```diff
@@ -0,0 +1,266 @@
+# Client Changes for Rekor v2
+
+This document outlines the changes clients need to make to support
+Rekor v2.
+
+## Rekor v2 API
+
+Rekor v2 supports HTTP or gRPC, like Fulcio. For Go, we have implemented a client
+already. For other languages, they can either use the OpenAPI docs or generated
+gRPC service proto, or create their own client.
+
+The service implements one write API, `/api/v2/log/entries`. Example JSON request bodies
+below:
+
+```json
```

Comment:
I haven't tested this, but `jsonc` appears to be a markdown rendering formation for json with comments.